### PR TITLE
issue #943

### LINKED ISSUE #943: Cross Site Access Secrets sections for Kubernetes and OpenShift are confusing each others
This issue was created from the discussion and situation analysis for this ticker #928

The following assumptions were found in the documentation which causes confusing and misleading
- sections 8.1.3, 8.1.4 and 8.1.5 follow each other, but seems to be customer should choose an options depends of target platform Kubernetes or OpenShift. It seems to be subsections for Kubernetes and OpenShift depends on customer selection, not plain sections list
- section 8.1.3 give only `certs` based option for the Kubernetes and 8.1.5 `token` based for the OpenShift, however it's a bit incorrect. `token` based option is available for the Kubernetes without access to the `certs` store. Getting `token` from the Kubernetes ServiceAccount also possible, but a bit different from the `oc sa get-token` command

### PR DESCRIPTION
**Pull request by oraNod**
Doc for fixes #943

### CODE REVIEW

File: documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 
-.Procedure
+Do one of the following:
 
-. Create secrets on each site that contain `ca.crt`, `client.crt`, and `client.key` from your Kubernetes installation.
+* Retrieve service account tokens from each site and then add them to secrets on each backup location, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ kubectl get serviceaccounts site-a -o yaml
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc:11]: Step should be the following
kubectl create serviceaccount site-a -n ns-site-a
kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=ns-site-a:site-a
TOKENNAME=`kubectl get serviceaccount/site-a -o jsonpath='{.secrets[0].name}' -n ns-site-a`
TOKEN=`kubectl get secret $TOKENNAME -o jsonpath='{.data.token}' -n ns-site-a | base64 --decode`

File: documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
+$ kubectl create secret generic site-a-secret --from-literal=token=<token>
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc:12]:
```suggestion
$ kubectl create secret generic site-a-secret -n ns-site-a --from-literal=token=$TOKEN
```

**Comment by dmvolod** [comment]
Overall very good, thanks a lot @oraNod 
Please see comments for details, I worried about the structure more

File: documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/asciidoc/topics/proc_creating_kubernetes_site_secrets.adoc]: thank you @dmvolod

**Review by dmvolod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks for capturing that feedback from community and reporting this issue @dmvolod I think it was very well spotted and is a great example of the precision that you bring to the team. Cheers! :tada: :tanabata_tree:

**Pull request merged**